src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ElementType } from 'react'
+import BootstrapBreadcrumb from 'react-bootstrap/Breadcrumb'
+
+interface Props {
+  as?: ElementType
+  label?: string
+  /** Defines the attributes of the children items */
+  itemProps?: {
+    as?: ElementType
+    href?: string
+    active?: boolean
+    target?: string
+    title?: string
+  }
```

Comment:
I think that we might be able to simplify this interface by doing: 

```ts
interface Props {
	label?: string
	children?: React.ReactNode
}
```

I think that we can use the defaults for everything else. 

What do you think?

Reply:
Thanks for taking a look. Using defaults sounds better to me.